### PR TITLE
Stop using tool/ci.sh in GitHub actions

### PR DESCRIPTION
**Pull request by kevmoo**
Don't create it if just using GitHub actions
Fewer moving pieces - less to maintain, etc
Also now using working-directory

### CODE REVIEW

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -36,12 +36,12 @@ jobs:
...
-        run: "cd mono_repo && pub upgrade --no-precompile"
+        working-directory: mono_repo
+        run: pub upgrade --no-precompile
       - name: "mono_repo; cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate"
-        env:
-          PKGS: mono_repo
-        run: tool/ci.sh command
         if: "steps.mono_repo_pub_upgrade.conclusion == 'success'"
+        working-directory: mono_repo
+        run: "cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate"
```
Comment (natebosch) [inline .github/workflows/dart.yml:44]: Am I misremembering or did @jakemac53 add support for top level repo steps? Should this get switched?

**Review by natebosch — APPROVED** [review]

File: .github/workflows/dart.yml (hunk 1 above)
Comment (kevmoo) [reply .github/workflows/dart.yml:44]: Not yet, sadly

**Review by jakemac53 — COMMENTED** [review]
Does the presubmit command still work with this change? I think that relies on the ci.sh script existing.

**Comment by kevmoo** [comment]
> Does the presubmit command still work with this change? I think that relies on the ci.sh script existing.

You mean the test? I think so. I'm getting rid of the `FORCE_PUB_COMMAND` stuff

**Comment by jakemac53** [comment]
> You mean the test? I think so. I'm getting rid of the `FORCE_PUB_COMMAND` stuff

I mean actually running `mono_repo presubmit`

**Comment by kevmoo** [comment]
> > You mean the test? I think so. I'm getting rid of the `FORCE_PUB_COMMAND` stuff
> 
> I mean actually running `mono_repo presubmit`

I think so...we'll see!

**Comment by jakemac53** [comment]
> I think so...we'll see!

The test probably also has travis enabled though, so it won't excersize this.

**Review by jakemac53 — APPROVED** [review]

**Pull request merged**